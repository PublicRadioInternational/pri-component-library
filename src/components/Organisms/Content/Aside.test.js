/**
 * @file Aside.test.js
 * Contains tests for Aside.component.js.
 */

import React from 'react';
import renderer from 'react-test-renderer';

import Aside from './Aside.component';

describe('<Aside />', () => {
  it('Matches the Aside snapshot', () => {
    const component = renderer.create(<Aside />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
