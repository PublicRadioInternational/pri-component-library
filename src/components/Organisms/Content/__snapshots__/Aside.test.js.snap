// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aside /> Matches the Aside snapshot 1`] = `
<aside
  className="className && null"
/>
`;
